crypto/incremental_patricia.re

```diff
@@ -33,6 +33,7 @@ module Make =
 
   let is_set = (bit, number) => 1 lsl bit land number != 0;
 
+  // Hash: 0e5751c026e543b2e8ab2eb06099daa1d1e5df47778f7787faab45cdf12fe3a8
```

Comment:
What's the purpose of this?

Reply:
It's to aid debugging. It's useful to know the hash of our hard-coded states so you can recognize them if you print them while debugging.